### ISSUE TITLE
[Accessibility: Resize text] Unable to edit all Template information when zoom 200%

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#898]
Only General, Storage and Interface tabs are displayed.

![124ed90bbf70e34f](https://cloud.githubusercontent.com/assets/4921718/14020098/4d46c50a-f1b4-11e5-96f3-1fa5e073b3f7.jpg)
